src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +191,26 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    /**
+     * {@code newDisplayName} is verified depending on {@code newIsDisplayedToStudents}.
```

Comment:
@leeyimin Have deleted the existing first line of the Javadoc comment (as it is basically repeated in the next two lines) and replaced it with the general statement in 0198869